src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -75,7 +75,7 @@
     <DocumentationFile>$(AssemblyName).xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Azure.Identity" Version="1.10.2" />
+    <PackageReference Include="Azure.Identity" Version="1.11.0" />
```

Comment:
Do we need to update the 1.10.2 version further down as well?

Reply:
we do, thanks for catching that: https://github.com/Azure/azure-functions-durable-extension/pull/2795/commits/2bbf50c18929c6a137c82b149339fcb9e1a80113